src/set.coffee

```diff
@@ -27,6 +27,7 @@ install = (_, jasmine) ->
   context = @
 
   globalPatches =
+    __autoIncrement: -1
```

Comment:
Anyways, this should start at `0` right? Since otherwise `globalPatches.__autoIncrement++` will return `-1` when first invoked, which I don't think we want